infrastructure/terraform.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  backend "s3" {
+    role_arn = "arn:aws:iam::756629837203:role/catalogue-developer"
+
+    bucket         = "wellcomecollection-catalogue-infra-delta"
+    key            = "terraform/concepts-pipeline/concepts_pipeline.tfstate"
+    dynamodb_table = "terraform-locktable"
+    region         = "eu-west-1"
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.22.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "eu-west-1"
+
+  assume_role {
+    role_arn = "arn:aws:iam::756629837203:role/catalogue-developer"
+  }
```

Comment:
Can we toss in some default tags here, at minimum a TerraformConfigurationURL?

Reply:
Ah yes, I always forget about those!